ST_mortality.c

```diff
@@ -489,17 +537,24 @@ static void _pat( const SppIndex sp) {
 }
 
 
-/***********************************************************/
-static void _mound( const SppIndex sp) {
-/*======================================================*/
-/* Ant mounds kill all but the hardiest plants. In C&L-90
- * that would be the succulents.
+/**
+ * \brief Implements mortality by ant mounds.
+ * 
+ * This function is called my mort_Main(). If there is a mound on the plot
+ * it kills most species
```

Comment:
add:

with the exception of succulents (Coffin and Lauenroth 1990)

Reply:
Addressed by commit 984bd13.